lib/tasks/task.rake

```diff
@@ -0,0 +1,214 @@
+namespace :allrecipe do
+  task :init, [:sFrequency] => :environment do |task, args|
+    get_ar_url(args.sFrequency.to_i)
+  end
+
+  task :image_collection => :environment do 
+    gather_img_urls()
+  end
+
+  def gather_img_urls()
+    require 'open-uri'
+    require 'nokogiri'
+
+    recipes = Recipe.all
+    recipes.each do |recipe|
+      if recipe.
+    end
+  
+  end
+
+  def get_ar_url(sFrequency)
+    require 'open-uri'
+    require 'nokogiri'
+    eFrequency = sFrequency + 9
```

Comment:
What are eFrequency and sFrequency? If you need to explain me, it means that these variable names are not self-explanatory enough